### PR TITLE
Fix a pile of warnings Java is complaining about.

### PR DESCRIPTION
**Pull request by brian-brazil**

### CODE REVIEW

File: src/com/carolinarollergirls/scoreboard/utils/ClockConversion.java [hunk 1]
```diff
@@ -17,11 +17,11 @@ public static Long fromHumanReadable(String v) {
                 par = Long.parseLong(parPad.substring(0, 3));
             }
 
-            return new Long(((sec + (min * 60)) * 1000) + par);
+            return Long.valueOf(((sec + (min * 60)) * 1000) + par);
         }
 
         try {
-            return new Long(Long.parseLong(v));
+            return Long.valueOf(Long.parseLong(v));
```
Comment (frank-weinberg) [inline src/com/carolinarollergirls/scoreboard/utils/ClockConversion.java]: valueOf() can directly take a String argument.
And I think many of the other instances can be removed due to autoboxing.

Comment (brian-brazil) [reply src/com/carolinarollergirls/scoreboard/utils/ClockConversion.java]: Updated. I'd done them in batch.

We should really get off the hand-edited JSON library...

Comment (frank-weinberg) [reply src/com/carolinarollergirls/scoreboard/utils/ClockConversion.java]: > We should really get off the hand-edited JSON library...

As we should probably update to upstream supported versions of most of our dependencies.

Comment (brian-brazil) [reply src/com/carolinarollergirls/scoreboard/utils/ClockConversion.java]: Indeed, it's somewhere on my todo list.

**Pull request merged**